Task1/public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
same for this file, `index.html` is in `public/` folder already. I would suggest you merge them, and remove folder Task1/